profiler/include/profiler/profile_pool3d_fwd_impl.hpp

```diff
@@ -233,13 +232,18 @@ bool profile_pool3d_fwd_impl(MaxPoolFwdInputParams& in_params,
             using BF16     = ck::bhalf_t;
             using F8       = ck::f8_t;
             auto tolerance = 1e-3;
-            if(std::is_same<InDataType, BF16>::value && std::is_same<OutDataType, BF16>::value)
+            if constexpr(ReduceOpId != ck::ReduceTensorOp::MAX)
             {
-                tolerance = 1e-2;
-            }
-            else if(std::is_same<InDataType, F8>::value && std::is_same<OutDataType, F8>::value)
-            {
-                tolerance = 1e-1;
+                if constexpr(std::is_same<InDataType, BF16>::value &&
+                             std::is_same<OutDataType, BF16>::value)
+                {
+                    tolerance = 1e-2;
```

Comment:
Let's make sure that this range of error is expected.

Reply:
Fixed mismatch in latest commit